model-optimizer/extensions/middle/ReverseV2ToReverseSequence.py

```diff
@@ -32,29 +35,51 @@ def pattern():
 
     def replace_pattern(self, graph: Graph, match: dict):
         reverse = match['reverse']
-        input_data_shape = reverse.in_node(0).shape
-
-        if len(input_data_shape) == 1:
-            raise Error('Reverse operation name = {} is\'t supported because of 1D input.'.format(reverse.name))
+        reverse_name = reverse.soft_get('name', reverse.id)
 
```

Comment:
I think we need to check if `reverse['axis']` is empty. If yes, then we need to remove this node.
Also, I think we need to check data in `reverse['axis']` and normalize it.